VATSpy.dat

```diff
@@ -17863,7 +17863,7 @@ LWSS|Skopje||LWSS
 LYBA|Beograd||LYBA
 LZBB|Bratislava||LZBB
 MDCS|Santo Domingo||MDCS
-MHTG|Central America||MHTG

```

Comment:
Just clarifying something about the position name with them; should be fine to merge after

Reply:
New position's (MHCC_CTR) radio callsign is "Cenamer Centre"